### DISCUSSION TITLE
Definition/use of GO:0040035 hermaphrodite genitalia development

### DISCUSSION THREAD

**Discussion by suzialeksander** [geneontology/helpdesk#399] [General]
From Helpdesk email: 

> Dear curators,
> 
> Thank you for providing this excellent resource which I have been using for almost 15 years now. After this long period, I finally encountered a problem with Gene Ontology. In a recent manuscript, we reported some results of a Gene Ontology analysis for C. elegans genes. One of the terms that came up was 'hermaphrodite genitalia development' (GO:0040035).
> 
> One of the reviewers remarked:
> 
> "NB (page 7): Nematodes don’t have hermaphrodite genitalia. Hermaphroditic nematodes have female genitalia. 
> This looks like someone else’s terminology, but I think the wording could still be upgraded here."
> 
> I asked several colleagues and they all agreed with the reviewer's view.
> 
> Therefore, I would like to bring your attention to this issue and I would suggest to assess, whether a change in the Gene Ontology would be possible and appropriate. If you need another expert opinion, I would recommend to ask Paul Sternberg who is also listed as PI on the GO website.

**Comment by suzialeksander** [comment]
This email came in earlier this month and I didn't see a reply or active ticket so I'm assuming it was not yet addressed.  @vanaukenk I'm not sure if this is annotation instead of ontology??

**Comment by vanaukenk** [comment]
Thanks @suzialeksander 
This looks like an ontology issue.  I'll take a look.

**Reply by cmungall** [reply]
it's in wbbt (which is linked indirectly via go->uberon->wbbt)

https://www.ebi.ac.uk/ols/ontologies/wbbt/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FWBbt_0005178&lang=en&viewMode=PreferredRoots&siblings=false

**Comment by suzialeksander** [comment]
Closed as issue is transferred to https://github.com/geneontology/go-ontology/issues/23531
